lib/Elastica/Aggregation/AdjacencyMatrix.php

```diff
@@ -10,16 +10,10 @@ class AdjacencyMatrix extends AbstractAggregation
     /**
      * Add a named filter.
      *
-     * @param string $name
-     *
      * @return $this
      */
-    public function addFilter(AbstractQuery $filter, $name): self
+    public function addFilter(AbstractQuery $filter, string $name): self
```

Comment:
Hum interesting, now with the type hint, the arguments are casted to string, as we don't use `declare(strict_types=1);` it doesn't throw an exception. Only here as there was no type-hint before.

I'm not sure if the test is still relevant?

Reply:
Removing the test for me is probably ok. Need to see the code but should be fine.

Reply:
I agree, when moving to ES-7 I also removed tests that did not make sense anymore when we had typehints